File: chat-service/src/main/java/org/example/config/SwaggerConfig.java
package org.example.config;

import io.swagger.v3.oas.models.OpenAPI;
import io.swagger.v3.oas.models.info.Info;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class SwaggerConfig {
    
    @Bean
    public OpenAPI openAPI() {
        return new OpenAPI()
                .info(new Info()
                        .title("채팅 서버 API 문서")
                        .description("채팅 서버 API 명세서")
                        .version("v1.0.0"));
    }
} 

File: chat-server-kotiln/src/main/kotlin/com/chat/chatserverkotiln/module/swagger/SwaggerConfig.kt
package com.chat.chatserverkotiln.module.swagger

import io.swagger.v3.oas.annotations.OpenAPIDefinition
import io.swagger.v3.oas.annotations.info.Contact
import io.swagger.v3.oas.annotations.info.Info // ✅ 어노테이션용
import org.springframework.context.annotation.Configuration

@OpenAPIDefinition(
    info = Info(
        title = "Chat API",
        version = "v1",
        description = "WebFlux 기반 채팅 서비스 API",
        contact = Contact(name = "상규", email = "sanggyu@example.com")
    )
)
@Configuration
class SwaggerConfig {
}